torch/quantization/observer.py

```diff
@@ -516,6 +516,49 @@ def calculate_qparams(self):
 
         return scale.to(dtype=torch.float), torch.tensor([nudged_zero_point])
 
+class _MinMaxTensorListObserver(MinMaxObserver):
```

Comment:
do you need to do this for all observers?

Reply:
Could you add a comment here talking about the context? that we will want to refactor if we want to support other types of observer?